File: 215-kth-largest-element-in-an-array/215-kth-largest-element-in-an-array.java
class Solution {
    public int findKthLargest(int[] nums, int k) {
        PriorityQueue<Integer> hp = new PriorityQueue<>();
        for(int i: nums){
            hp.add(i);
            if(hp.size()>k) hp.poll();
        }
        return hp.peek();
    }
}

File: 215-kth-largest-element-in-an-array/215-kth-largest-element-in-an-array.py
class Solution:
    def findKthLargest(self, nums: List[int], k: int) -> int:
        h=[]
        for i in range(len(nums)):
            heapq.heappush(h, nums[i]);
            if len(h)>k:
                heapq.heappop(h)
        return h[0]